app/assets/javascripts/shared/dradis_datatable.js

```diff
@@ -64,6 +64,26 @@ class DradisDatatable {
       },
       lengthChange: false,
       pageLength: 25,
+      stateSave: true,
+      stateSaveCallback: function(settings, data) {
+        var dataTableDomId = settings.oInstance[0].id;
+        var localStorageKey = `Datatables_${dataTableDomId}_${window.location.pathname}`;
+
+        // Check if the table has been loaded before so that we can skip
+        // the manually hidden columns on page load.
+        var initialLoad = false;
+        if (localStorage.getItem(localStorageKey) === null) {
+          initialLoad = true;
+        }
+
+        data.initialLoad = initialLoad;
+        localStorage.setItem(localStorageKey, JSON.stringify(data));
+      },
+      stateLoadCallback: function(settings) {
```

Comment:
If you'll move the variables to the class, we can simplify this function to a single line, which I don't think requires it be a class method as it'll be simple enough.

Reply:
Refactored it into method instead